### PR TITLE
fix(Bloom): add theme manifest

### PR DESCRIPTION
**Pull request by nimsandu**

### CODE REVIEW

**Comment by nimsandu** [comment]
@afonsojramos sir? kinda excited to see the theme on marketplace !

**Review by nimsandu — COMMENTED** [review]
including bloom theme on manifeast.json

File: manifest.json [hunk 1]
```diff
@@ -102,5 +102,21 @@
...
\ No newline at end of file
+    },
+    {
+    "name": "Bloom",
+    "description": "Spicetify theme inspired by Microsoft's Windows 11 Bloom Experience",
+    "preview": "https://raw.githubusercontent.com/nimsandu/spicetify-bloom/main/Dark-1.png",
+    "readme": "README.md",
+    "usercss": "user.css",
+    "schemes": "color.ini",
+    "include": ["https://raw.githubusercontent.com/nimsandu/spicetify-bloom/main/bloom.js"],
```
Comment (afonsojramos) [inline manifest.json:113]:
```suggestion
    "include": ["https://raw.githubusercontent.com/spicetify/spicetify-themes/master/Bloom/bloom.js"],
```

I'd prefer if you used the script in this repo.

File: manifest.json [hunk 2]
```diff
@@ -102,5 +102,21 @@
...
         ]
-    }
-]
\ No newline at end of file
+    },
+    {
+    "name": "Bloom",
+    "description": "Spicetify theme inspired by Microsoft's Windows 11 Bloom Experience",
+    "preview": "https://raw.githubusercontent.com/nimsandu/spicetify-bloom/main/Dark-1.png",
+    "readme": "README.md",
```
Comment (afonsojramos) [inline manifest.json:110]:
```suggestion
    "readme": "Bloom/README.md",
```

**Review by afonsojramos — CHANGES_REQUESTED** [review]

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**